Documentation program

UnCool-0x opened the pull request.

pranay01 commented:
@prashant-shahi Can we find a better place for it in the docs structure rather than the top level. Doesn't see like that big a topic to put a new top level section IMO

![CORS](https://user-images.githubusercontent.com/504541/226450906-f75e3d88-7fc0-4ee2-bc5a-575fcbbaf8b5.png)

prashant-shahi commented:
> @prashant-shahi Can we find a better place for it in the docs structure rather than the top level. Doesn't see like that big a topic to put a new top level section IMO

@pranay01 I thought of including it under apm/metrics/logs section but did not since it is relevant for all of them.
Alternative would be tutorials section for this.

ankit01-oss commented:
Let's do it under `Configurations`? @pranay01 @prashant-shahi 

This is important for browser instrumentation too. We have then instructions mentioned in our blogs for angular and react.

prashant-shahi commented:
@pranay01 @ankit01-oss Moved the page to Configuration section.

harshbutfairx committed:
feat: add delayed close functionality to product and resources dropdowns (#792)

yuvraajsj18 committed:
launch week thurs (#785)

* update

* update

* merged

* add blog

* add blog

---------

Co-authored-by: Ankit Anand <83692067+ankit01-oss@users.noreply.github.com>

yuvraajsj18 committed:
ankit chat blog (#793)

* update

* update

* merged

* add

ankit01-oss committed:
Changes for day 4 (#794)

Co-authored-by: ankit01-oss <>

nityanandagohain committed:
feat: docs for logs-v2 (#790)

* feat: docs for logs-v2

* feat: minor fix

yuvraajsj18 committed:
launch week fri (#786)

* update

* update

* merged

* add blog

ankit01-oss committed:
Changes for day 5 (#796)

Co-authored-by: ankit01-oss <>

harshbutfairx committed:
fix: fix youtube playback error (#798)

yuvraajsj18 committed:
fix issues in otel nodejs (#799)

* update

* update

* merged

* minor fixes

ankit01-oss committed:
Removed launch week banner and add launch week link in footer (#801)

Co-authored-by: ankit01-oss <>

jasalt committed:
Fix typos in collect_logs_from_file.mdx (#802)

konnta0 committed:
fix: typo Azure Blob Storage SideNav (#800)

harshbutfairx committed:
fix: layout fix for cards (#803)

yuvraajsj18 committed:
fix feed xml (#791)

* update

* update

* merged

* fix feed.xml invalid link issue

ankit01-oss committed:
Added case study for Cedana (#809)

nityanandagohain committed:
feat: migration guide for clickhouse queries (#812)

yuvraajsj18 committed:
upload articles 25 09 (#811)

* update

* update

* merged

* update

* update

* Add Articles

* add more

nityanandagohain committed:
feat: migration guide 0.55 (#813)

* feat: migration guide 0.55

* fix: minor fixes

* fix: minor fixes

* fix: minor fixes

* fix: minor fixes

prashant-shahi committed:
chore: minor v0.55 migration changes (#814)

anazworth committed:
Fix typos on PostgreSQL integrations and otel collector docs (#773)

yuvraajsj18 committed:
Scroll Form FAQs (#797)

* update

* update

* merged

* trying

* Add what you looking for form on home

* added form to faq pages

* fix issue

* final design

* Update options

ankit01-oss committed:
Case study with Kiwi (#815)

Calm-Rock committed:
update: updated vector logs docs (#807)

* update: updated vector logs docs

* fix: removing yarn file

* fix: removed yarn lock file

Calm-Rock committed:
fix: fixed the feedback component location (#818)

ankit01-oss committed:
Minor edit (#819)

Co-authored-by: ankit01-oss <>

skeptrunedev committed:
feat: add Trieve search and AI Chat (RAG) (#816)

Co-authored-by: Sara Vieira <hey@iamsaravieira.com>

nityanandagohain committed:
fix: add links in migration guide (#822)

Calm-Rock committed:
fix: removed info block from self host instructions (#824)

ankit01-oss committed:
Added case study for bands (#825)

* Added case study for bands

* removed image

* added image

* Update data/case-study/bands.mdx

---------

Co-authored-by: ankit01-oss <>

UnCool-0x committed:
feat: mobile-instrumentation (#808)

* feat: mobile-instrumentation

* minor changes

* notice

* new tag

* feat: removed self hosted signoz

* fix github urls

* feat: removed new doc tag

* feat: region

* feat: removed smaller table

Calm-Rock committed:
fix: removed new tag from mobile monitoring (#827)

yuvraajsj18 committed:
fix structure data (#828)

* update

* update

* merged

* update

* update

* fix

* fix feed.xml

harshbutfairx committed:
feat: add search button to mobile (#830)

* feat: add search button to mobile

* fix: add gap between search and hamburger

* fix: search button hidden when drop down is active

Calm-Rock committed:
fix: moved mobile monitoring section (#832)

pranay01 committed:
added BAA in teams plan (#834)

sharpshooter90 committed:
feat(perf): optimized large assets (#835)

- removed un used fonts for new theme
- removed un used CSS
- optimized image asset worth ~3mb

UnCool-0x committed:
chore: merge

UnCool-0x committed:
chore: lock file
